### PR TITLE
refactor(Less): removed deprecated less variables

### PR DESCRIPTION
**Pull request by kevincareer427**
JIRA: https://jira.rax.io/browse/FRMW-751

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] Design LGTM

### CODE REVIEW

**Comment by kevincareer427** [comment]
jira: https://jira.rax.io/browse/FRMW-751

**Comment by kevincareer427** [comment]
Migration Guide is updated:
https://github.com/rackerlabs/encore-ui/wiki/1.x-Deprecated-Items#less-variables

**Comment by kevincareer427** [comment]
## Before
before

## After
after

File: src/components/layout/responsiveLayout.less [hunk 1]
```diff
@@ -19,21 +18,23 @@
...
+    background-color: @green-700;
   }
   .light-green {
-    background-color: @light-green;
+    background-color: @green-500;
   }
   .orange {
-    background-color: @orange;
+    background-color: @orange-500;
+    color: @gray-900;
```
Comment (CITguy) [inline src/components/layout/responsiveLayout.less]: please bump this to a `@gray-950` to match the `.dark-text` styling used in `demo/less/_palette.less`

Comment (kevincareer427) [inline src/components/layout/responsiveLayout.less]: @CITguy  updated

**Comment by 100stacks** [comment]
@kevincareer427, I added the JIRA story for you in your initial comment.  You can edit comments by selecting the ✏️ in the upper right hand corner.

screen shot 2016-05-19 at 4 00 01 pm

**Comment by 100stacks** [comment]
Did the visual regression trigger on this PR? If not, can you please add back the screenshots?  Thanks!

**Comment by kevincareer427** [comment]
## Before
before

## After
after

**Comment by kevincareer427** [comment]
@100stacks  Thanks! screenshots added.

File: demo/less/_palette.less [hunk 2]
```diff
@@ -1,80 +1,3 @@
...
-    background-color: @orange;
-    color: @white;
-  }
-
-  .swatch-red {
-    background-color: @red;
-    color: @white;
-  }
-}//.swatch
-
```
Comment (CITguy) [inline demo/less/_palette.less]: 👍

File: demo/templates/styles/color-1-0.html [hunk 3]
```diff
@@ -1,104 +0,0 @@
...
-          <li>
-            Used for link colors and icons associated with destructive/destroy
-            actions. Irreversible changes (like rebuilding a server) are not
-            considered destructive actions.
-          </li>
-        </ul>
-      </div>
-    </div>
-  </section>
-</rx-page>
```
Comment (CITguy) [inline demo/templates/styles/color-1-0.html]: 👍

File: demo/templates/styles/color.html [hunk 4]
```diff
@@ -8,11 +8,6 @@
     hard-coded hex codes in the future. Hex codes are provided here
     for reference only.
   </p>
-  <p>
-    We have <a href="#/styles/color-1-0">a reference to the old 1.0 color palette</a>
-    for your convenience, but note that the 1.0 palette is slated for deprecation
-    and shouldn't be used. Any styles that use the 1.0 palette should be updated.
-  </p>
```
Comment (CITguy) [inline demo/templates/styles/color.html]: 👍

File: src/components/layout/responsiveLayout.less [hunk 5]
```diff
@@ -4,7 +4,6 @@
     color: white;
     font-weight: bold;
     padding: 20px;
-    text-shadow: 0 0 4px rgba(100, 100, 100, 1);
```
Comment (CITguy) [inline src/components/layout/responsiveLayout.less]: 👍

File: src/components/layout/responsiveLayout.less [hunk 6]
```diff
@@ -19,21 +18,23 @@
...
+    background-color: @green-700;
   }
   .light-green {
-    background-color: @light-green;
+    background-color: @green-500;
   }
   .orange {
-    background-color: @orange;
+    background-color: @orange-500;
+    color: @gray-950;
```
Comment (CITguy) [inline src/components/layout/responsiveLayout.less]: 👍

File: src/components/rxBulkSelect/docs/rxBulkSelect.demo.less [hunk 7]
```diff
@@ -1,11 +1,11 @@
...
   }
   .success-status {
-    color: @green;
+    color: @green-700;
   }
   .failure-status {
-    color: @red;
+    color: @red-500;
   }
 }
```
Comment (CITguy) [inline src/components/rxBulkSelect/docs/rxBulkSelect.demo.less]: These actually aren't being applied, but we'll deal with that issue later. 👍

**Comment by CITguy** [comment]
LGTM

File: src/components/layout/responsiveLayout.less (hunk 5 above)
Comment (100stacks) [inline src/components/layout/responsiveLayout.less]: I wondered why the text looked fuzzy when different colors were applied...now I know this was culprit. 💯

**Comment by 100stacks** [comment]
@kevincareer427, no problem! LGTM 👍

**Comment by glynnis** [comment]
LGTM

**Pull request merged**